sn_protocol/src/messages/query.rs

```diff
@@ -29,6 +29,13 @@ pub enum Query {
     /// [`Chunk`]:  crate::storage::Chunk
     /// [`GetChunk`]: super::QueryResponse::GetChunk
     GetChunk(ChunkAddress),
+    /// Retrieve a [`Register`] at the given address.
+    ///
+    /// This should eventually lead to a [`GetRegister`] response.
+    ///
+    /// [`Register`]: sn_registers::Register
+    /// [`GetRegister`]: super::QueryResponse::GetRegister
+    GetRegister(RegisterAddress),
```

Comment:
It's for signed or un-signed?

Reply:
It's signed, corrected the comments as Roland noted above